index.js

```diff
@@ -31,7 +31,10 @@ exports.searchFirestore = functions.firestore
             var userCollection = firestore.collection(collectionName);
             if (where) {
                 where.forEach(function(subquery) {
-                    userCollection = userCollection.where(subquery.field, subquery.operator, subquery.val);
+                    if (subquery.val)
```

Comment:
nit: can we use braces here? For both `if` and `else` statements. (Context: I try to follow the [Google JS Style Guide](https://google.github.io/styleguide/jsguide.html#formatting-braces-all))

Reply:
Of course it can be so.